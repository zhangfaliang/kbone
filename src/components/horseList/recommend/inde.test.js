import toJson from 'enzyme-to-json';
import { render, mount } from 'enzyme';
import Recommend from './index';

describe('recommend', async () => {
  let data = { title: 222, list: [], dataList: [] };
  it('Recommend', () => {
    expect.assertions(1);
    const wrapper = render(
      <Recommend
        data={data}
        keys={{ title: 'location', information: 'list' }}
        item={() => {}}
        src="11111"
      />
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
