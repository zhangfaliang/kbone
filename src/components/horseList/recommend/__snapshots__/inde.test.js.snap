// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`recommend Recommend 1`] = `
Array [
  <div
    class="title"
  >
    <span>
      <img
        alt="Horse Racing Logo"
        src="11111"
      />
    </span>
    <span />
  </div>,
  <div
    class="container"
  />,
]
`;
